src/tox/config/cli/env_var.py

```diff
@@ -17,9 +17,7 @@ def get_env_var(key: str, of_type: Type[Any]) -> Optional[Tuple[Any, str]]:
     :param of_type: the type we would like to convert it to
     :return:
     """
-    key_upper = key.upper()
-    for fmt in ("TOX_{}", "TOX{}"):
-        environ_key = fmt.format(key_upper)
+    for environ_key in (f"TOX_{key.upper()}", f"TOX{key.upper()}"):
```

Comment:
technically here you're calling upper twice, you could use a variable as was done before to do it only once